app/views/pages/documentation.html.erb

```diff
@@ -98,6 +98,12 @@
     <%= image_tag("AUK_neighbours.png", alt: "AUK Interactive Hyperlink Diagram Explained", class:"body_img")%>
     <%= image_tag("AUK_hover.png", alt: "AUK Interactive Hyperlink Diagram hover feature", class:"body_img")%>
 
+    <a name="notebooks"></a><h3 class="about_h3">Exploring Derivatives with the Archives Unleashed Cloud Jupyter Notebooks</h3>
+    <%= image_tag("AUK_Notebook.png", alt: "AUK Jupyter Notebook", class:"body_img")%>
+    <p class="about_p">We have prototype Jupyter Notebooks available for you to work with the derivatives generated by the Archives Unleashed Cloud. You can read more about this in our blog post "<%= link_to('Exploring Web Archival Data through Archives Unleashed Cloud Jupyter Notebooks', 'https://news.archivesunleashed.org/exploring-web-archival-data-through-archives-unleashed-cloud-jupyter-notebooks-7605c6ca2b33', target: '_blank') %>". They allow you to interactively explore and filter the domain count information, extracted full text, and network visualization data generated by the Archives Unleashed Cloud.</p>
+    <p class="about_p">We are currently exploring greater integration between the notebooks and the Archives Unleashed Cloud. To use them now, please visit the GitHub repository <%= link_to('here', 'https://github.com/archivesunleashed/auk-notebooks', target: '_blank') %> and follow the instructions.</p>
+    <p class="about_p">Note that this is a prototype service. Any and all feedback and suggestions are greatly appreciated.</p>
```

Comment:
drop 'service'

(maybe: 'Note that this is still in the prototype stage.')?

Reply:
Perhaps we should add in contact info for feedback? 

'Any and all feedback and suggestions are greatly appreciated and can be sent to our project manager at sam.fritz@archivesunleashed.org'

Reply:
Great idea, let me add this!